src/clj/pb/routes/api.clj

```diff
@@ -0,0 +1,66 @@
+(ns pb.routes.api
+  (:require [pb.db.core :refer [*db*] :as db])
+  (:require [pb.layout :as layout]
+            [compojure.core :refer [context defroutes GET POST]]
+            [ring.util.http-response :as response]
+            [clojure.java.io :as io]
+            [buddy.hashers :as hashers]
+            [clojure.java.jdbc :as jdbc]
+            [clojure.string :as str]
+            [twilio.core :as twilio]))
+
+(defonce PB_TWILIO_AUTH_TOKEN (System/getenv "PB_TWILIO_AUTH_TOKEN"))
+(defonce PB_TWILIO_ACCOUNT_SID (System/getenv "PB_TWILIO_ACCOUNT_SID"))
+(defonce PB_TWILIO_PHONE_NUMBER (System/getenv "PB_TWILIO_PHONE_NUMBER"))
+
+(defn db-tx [f & [args]]
```

Comment:
we should probably add a `try / catch` around this function, it case it ever throws. We know where its throwing from instead of the usual 100+ line stack traces in java 😓 (esp when we deploy to prod somewhere this will come in handy)